docs/nuget-org/Publish-a-package.md

```diff
@@ -1,122 +1,125 @@
 ---
-title: How to Publish a NuGet Package
-description: Detailed instructions for how to publish a NuGet package to nuget.org or private feeds, and how to manage package ownership on nuget.org.
+title: How to publish NuGet packages
+description: See detailed instructions about how to publish a NuGet package and manage package ownership on nuget.org.
 author: JonDouglas
 ms.author: jodou
-ms.date: 05/18/2018
+ms.date: 8/29/2022
 ms.topic: conceptual
 ms.reviewer: anangaur
 ---
 
-# Publishing packages
+# Publish NuGet packages
 
-Once you have created a package and have your `.nupkg` file in hand, it's a simple process to make it available to other developers, either publicly or privately:
+Once you create a NuGet package and have a *.nupkg* file, you can make the package available to other developers either publicly or privately. This article describes how to share public packages globally through [nuget.org](https://www.nuget.org/packages/manage/upload).
 
-- Public packages are made available to all developers globally through [nuget.org](https://www.nuget.org/packages/manage/upload) as described in this article (requires NuGet 4.1.0+).
-- Private packages are available to only a team or organization, by hosting them either a file share, a private NuGet server, [Azure Artifacts](https://www.visualstudio.com/docs/package/nuget/publish), or a third-party repository such as myget, ProGet, Nexus Repository, and Artifactory. For additional details, see [Hosting Packages Overview](../hosting-packages/overview.md).
-
-This article covers publishing to nuget.org; for publishing to Azure Artifacts, see [Package Management](https://www.visualstudio.com/docs/package/nuget/publish).
+You can also make private packages available to only a team or organization by hosting them on a file share, a private NuGet server, or a third-party repository such as myget, ProGet, Nexus Repository, or Artifactory. For more information, see [Host your own NuGet feeds](../hosting-packages/overview.md). For publishing with [Azure Artifacts](https://www.visualstudio.com/docs/package/nuget/publish), see [Publish packages to NuGet.org](/azure/devops/artifacts/nuget/publish-to-nuget-org).
 
 ## Publish to nuget.org
 
-For nuget.org, you must sign in with a Microsoft account, with which you'll be asked to register the account with nuget.org.
+To publish on nuget.org, sign in to nuget.org with a Microsoft account, and use it to create a free nuget.org account. Follow the instructions at [Add a new individual account](individual-accounts.md#add-a-new-individual-account).
+
+![Screenshot that shows the NuGet sign in link.](media/publish_NuGetSignIn.png)
```

Comment:
media/publish_NuGetSignIn.png -> media/publish_NuGetSignIn.PNG?

Seems that the image path is broken.

Reply:
@zhhyu - It works for me, but the uppercase extension is nonstandard and has caused problems in other repos so I changed it (for both graphics). (Usually the only way to do this from Windows machines is to rename the graphics.)